operate/ledger/profiles.py

```diff
@@ -169,3 +171,16 @@
 }
 
 ERC20_TOKENS = [OLAS, USDC, WRAPPED_NATIVE_ASSET]
+
+
+def get_staking_contract(
+    chain: str, staking_program_id: t.Optional[str]
+) -> t.Optional[str]:
+    """Get staking contract based on the config and the staking program."""
+    if staking_program_id == NO_STAKING_PROGRAM_ID or staking_program_id is None:
```

Comment:
We should remove this in the future. staking_program_id must be None if there is none selected.